lib/spree/search/multi_domain.rb

```diff
@@ -1,13 +1,8 @@
 module Spree::Search
   class MultiDomain < Spree::Core::Search::Base
     def get_base_scope
```

Comment:
I was having a look at the [class extensions points](https://guides.solidus.io/developers/preferences/class-extension-points.html) section in Solidus guides, and I noticed it suggests `get_base_scope` should be a private method. Do you think this is necessary here or we can skip it altogether?